src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -51,6 +54,29 @@ describe('LibraryServiceClient', function() {
         done();
       });
     });
+
+    it('invokes createShelf with error', function(done) {
+      var client = libraryV1.libraryServiceClient();
+      // Mock request
+      var shelf = {};
+      var request = {
+          shelf : shelf
+      };
+
+      // Mock Grpc layer
+      client._createShelf = function(actualRequest, options, callback) {
```

Comment:
I don't have strong feeling here, both would look good to me.  To me, skipping expected response as `null` isn't very bad.